frontend/actions/actions.js

```diff
@@ -0,0 +1,20 @@
+const AppDispatcher = require("../dispatcher/dispatcher");
```

Comment:
@danielchangNYC Ah this is a great call Daniel.  I believe I could use`resolve.moduleDirectories`the way I resolved the extensions.